WordPress/wordpress-playground
# [Remote] Preserve PHP constants when saving a temporary site

This PR ensures that Blueprint-defined constants, such as WP_DEBUG,
are preserved in OPFS when a temporary site is saved, and are defined
again on subsequent boot after a page reload

 ## Testing plan

* Go to http://localhost:5400/website-server/?login=yes
* Confirm you are logged in
* Save the site in OPFS
* Reload the browser tab
* Confirm you are still logged in## Motivation for the change, related issues
